File: codeforces/low_difficulty/difficulty_800/73.boring_apartments_[1433A]/boring_apartments.py
# Author: José Rodolfo (jric2002)
t = int(input())
while (t):
  x = int(input())
  n = len(str(x))
  p = int(((n * (n + 1)) / 2) + (10 * ((x % 10) - 1)))
  print(p)
  t -= 1

File: codeforces/low_difficulty/difficulty_800/73.boring_apartments_[1433A]/boring_apartments.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int t, x, n, p;
  cin >> t;
  while (t--) {
    cin >> x;
    n = (to_string(x)).size();
    p = ((n * (n + 1)) / 2) + (10 * ((x % 10) - 1));
    cout << p <<endl;
  }
  return 0;
}